File: solutions/erlang/raindrops/1/src/raindrops.erl
-module(raindrops).

-export([convert/1]).


convert(Number) when Number rem 3 == 0, Number rem 5 == 0, Number rem 7 == 0 -> "PlingPlangPlong";
convert(Number) when Number rem 3 == 0, Number rem 5 == 0 -> "PlingPlang";
convert(Number) when Number rem 5 == 0, Number rem 7 == 0 -> "PlangPlong";
convert(Number) when Number rem 3 == 0, Number rem 7 == 0 -> "PlingPlong";
convert(Number) when Number rem 3 == 0 -> "Pling";
convert(Number) when Number rem 5 == 0 -> "Plang";
convert(Number) when Number rem 7 == 0 -> "Plong";
convert(Number) -> integer_to_list(Number).

File: solutions/lua/raindrops/1/raindrops.lua
return function(n)
  local a = ""
  if n % 3 == 0 then
    a = "Pling"
  end
  if n % 5 == 0 then 
    a = a .. "Plang"
  end
  if n % 7 == 0 then
    a = a .."Plong"
  end
  if a == "" then
    a = tostring(n)
  end
  return a
end

